app/renderer/components/task_suite.js

```diff
@@ -25,18 +25,23 @@ module.exports = {
             <div class="row" v-if="hideMessage">
                 <div class="grey-text text-lighten-1 section center-align" >You can add tasks by pressing the <i class="material-icons unselectable-text">mode_edit</i> button at the top</div>
             </div>
-            <draggable v-show="suite.tasks.length > 0 || AppStatus.editMode" element="ul" :options="{draggable:'.task-card'}" style="margin-bottom:0; margin-top:0 " class="collapsible" data-collapsible="accordion" v-model="draggableTasks" @end="restructureTasks" :move="checkMove">
-                <template v-for="(task,i) in suite.tasks">
-                    <task-card v-bind:task="task" v-on:restructureTasks="restructureTasks" v-on:remove="removeTask(i)" v-on:edit="editTask(i, $event)" v-bind:event="event"></task-card>
+            <draggable v-show="suites[index].tasks.length > 0 || AppStatus.editMode" element="ul" 
+            :options="{draggable:'.task-card', group:'tasks'}"
+            class="collapsible no-margin" 
+            data-collapsible="accordion" 
+            v-model="suites[index].tasks"
+            :move="checkMove">
+                <template v-for="(task,i) in suites[index].tasks">
+                    <task-card v-bind:task="task" v-on:remove="removeTask(i)" v-on:edit="editTask(i, $event)" v-bind:event="event"></task-card>
                 </template>
-            <add-task v-bind:tasks="suite.tasks" v-on:add="addTask" v-if="showAddTab"></add-task>
             </draggable>
+            <add-task v-bind:tasks="suites[index].tasks" v-on:add="addTask" v-if="showAddTab"></add-task>
         </div>
     `,
     mounted() {
         AppStatus.events.on("run-suite", this.onRunSuite);
         AppStatus.events.on("stop-suite", this.onStopSuite);
-        AppStatus.totalTasks += this.suite.length;
+        AppStatus.totalTasks += this.suites[this.index].length;
```

Comment:
I see you are using suites[this.index] a lot, probably is better to set a variable with the current suite?

I would use the `data()` method, returning the current suite